src/msw/mediactrl_am.cpp

```diff
@@ -1590,11 +1600,10 @@ void wxAMMediaEvtHandler::OnActiveX(wxActiveXEvent& event)
             // size of the video (will error) - however on 4
             // it won't play on downloaded things until it is
             // completely downloaded so we use the lesser of two evils...
-            else if(event[0].GetInteger() == 3 &&
+            else if(event[0].GetInteger() >= 3 &&
                 !m_bLoadEventSent)
             {
                 m_bLoadEventSent = true;
-                m_amb->FinishLoad();
```

Comment:
This really seems wrong, why are we setting `m_bLoadEventSent` to true if we're not actually sending the event?

Reply:
Calling m_amb->FinishLoad is sending the load event.